docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Resetting your password or reclaiming your username
+
+Whether you have to reset a forgotten password or remember the usernames associated with your {{brand}} account, you can do so via the {{gui}}.
+
+## Reset lost password
+
+Begin by navigating to the {{gui}}. At the bottom left-hand side of the page, click on the "Forgot your password?" button.
+
+![Forgot your password button](assets/Forgot-your-password.png)
+
+A pop-up window titled "Reclaim lost password!", appears. Type in your username, then click on the "Reclaim password!" button.
+
+![Reclaim password button](assets/Reclaim-lost-password.png)
+
+The content of the pop-up window changes, informing you that an email has been sent to you with instructions on how to reset your password.
+
+Click the "Ok!" button, then go ahead and check your inbox (or SPAM/junk folder) for an email with the subject **_Cleura Account password reset request_**.
```

Comment:
"Spam" is not an acronym, and therefore should not be in all caps. See https://en.wikipedia.org/wiki/Email_spam